offline_imu/plugin.cpp

```diff
@@ -18,10 +18,7 @@ class offline_imu : public ILLIXR::threadloop {
         , dataset_first_time{_m_sensor_data_it->first}
         , dataset_now{0}
         , imu_cam_log{record_logger_}
-        , _m_rtc{pb->lookup_impl<RelativeClock>()}
-        , _m_log{"metrics/offline_imu.csv"} {
```

Comment:
is this removal temporary?

Reply:
This logging seems like a one-time use case. No need to keep it around.